docs/binder_governance.rst

```diff
@@ -0,0 +1,135 @@
+Binder Project Governance
+=========================
+
+The following sections describe governance and high-level principles
+surrounding the Binder Project.
+
+The Binder Project is a member of Project Jupyter, which is a fiscally
+sponsored project of NumFocus, a US 501c3 non-profit.
+
+Project mission
+~~~~~~~~~~~~~~~
+
+The mission of the Binder Project is to create, advance, and promote
+open technology that makes it easy for people to connect their data
+science communication, education, and scientific reproducibility work
+with computational environments which can run their work.
```

Comment:
I find this language a bit challenging (e.g. what do you mean by "scientific reproducibility work"?) and I have a couple comments / suggestions about scope. 

-  I think that for many scientists who use binder, reproducibility of work may only be one goal - often disseminating their work to other researchers is the main goal and reproducibility is one requirement of that. I would suggest that the language be positioned more on "sharing work in a reproducible manner". 
- The current text suggests that the role of Binder is to connect computational work with a computational environment to run their work. I think this misses the fact that people are involved on both sides (someone can distributes their work using binder, and people can pick up and use someone else's work that has been shared).  

Here is a suggestion, feel free to adapt as you see fit.

```
The mission of the Binder Project is to create, advance, and promote 
open technology that makes it easy for people to connect their data 
science communications, educational materials, and scientific work
with computational environments where their work can be run and 
shared with others.   
```

Reply:
Copied over as is. I think it is much clearer this way.